src/styles/blocks/card.scss

```
@@ -0,0 +1,126 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  padding: 16px;
+  transition: transform 300ms;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 16px 0 40px;
+  }
+
+  &__name {
+    color: $dark-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    color: $light-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+    &-text {
+      color: $dark-text-color;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__stars {
+    display:flex;
+
+    @for $i from 1 through 5 {
+      &--#{$i} {
+        .card__star:nth-child(-n + #{$i}) {
+            background-image: url("../images/star-active.svg");
+        }
+      }
+    }
+  }
+
+  &__star {
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+    height: 16px;
+    width: 16px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $light-text-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $dark-text-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-decoration: none;
+    box-sizing: border-box;
+    padding: 12px;
+    border-radius: 5px;
+    border: 1px solid transparent;
+    background-color: $main-color;
+    color: $background-color;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 14px;
+    transition: 300ms;
+  }
+
+  &__button:hover {
+    border-color: $main-color;
+    background-color: $background-color;
+    color: $main-color;
+  }
```

use nesting for card__button:hover
```suggestion
    transition: 300ms;
    
    &:hover {
      border-color: $main-color;
      background-color: $background-color;
      color: $main-color;
    }
  }
```